client/client-multi/client-cpp/kaa/utils/ThreadPool.hpp

```diff
@@ -49,9 +47,15 @@ class ThreadPool : public IThreadPool {
     void start();
     void stop(bool force);
 
+    enum class State {
+        CREATED,
+        RUNNING,
+        PENDING_SHUTDOWN,
+        STOPPED
```

Comment:
General code style. It will be much easier to extend and shuffle values in the enum later.